### PR TITLE
Typo fixes in action cable.

### PR DESCRIPTION
**Pull request by alkesh26**
### Summary
1. Replaced unused variables with `_`.
2. Used `%w(..)` instead of `[...]`.
3. Typo fixes

### CODE REVIEW

File: actioncable/lib/action_cable.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@ module ActionCable
       server_restart: "server_restart"
     },
     default_mount_path: "/cable",
-    protocols: ["actioncable-v1-json", "actioncable-unsupported"].freeze
+    protocols: %w(actioncable-v1-json actioncable-unsupported).freeze
```
Comment (rafaelfranca) [inline actioncable/lib/action_cable.rb]: This change is not needed. Can you please revert?

File: actioncable/lib/rails/generators/channel/channel_generator.rb [hunk 2]
```diff
@@ -37,10 +37,7 @@ def file_name
         def generate_application_cable_files
           return if behavior != :invoke
 
-          files = [
-            "application_cable/channel.rb",
-            "application_cable/connection.rb"
-          ]
+          files = %w(application_cable/channel.rb application_cable/connection.rb)
```
Comment (rafaelfranca) [inline actioncable/lib/rails/generators/channel/channel_generator.rb]: This change is not needed. Can you please revert?

File: actioncable/test/stubs/global_id.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@ class GlobalID
   attr_reader :uri
   delegate :to_param, :to_s, to: :uri
 
-  def initialize(gid, options = {})
+  def initialize(gid)
```
Comment (rafaelfranca) [inline actioncable/test/stubs/global_id.rb]: We want to keep the same API so we want `options` being passed here.

File: actioncable/lib/action_cable.rb (hunk 1 above)
Comment (alkesh26) [reply actioncable/lib/action_cable.rb]: @rafaelfranca just curious to know why we prefer `[]` instead of `%w()`?

Comment (rafaelfranca) [reply actioncable/lib/action_cable.rb]: It is not a question of preference, it is that the code as it was is fine and we don't accept cosmetic changes in PRs.

File: actioncable/lib/rails/generators/channel/channel_generator.rb (hunk 2 above)
Comment (rafaelfranca) [reply actioncable/lib/rails/generators/channel/channel_generator.rb]: This one still needs to be reverted.

File: actioncable/lib/action_cable.rb (hunk 1 above)
Comment (alkesh26) [reply actioncable/lib/action_cable.rb]: cool

**Review by rafaelfranca — CHANGES_REQUESTED** [review]
Can you please squash your commits in one? Thanks

**Comment by alkesh26** [comment]
@rafaelfranca squashed the commits. Thanks for quick review.

**Pull request merged**